Makefile

```diff
@@ -235,21 +235,11 @@ test-cmd-devfile-exec:
 test-cmd-docker-devfile-exec:
 	ginkgo $(GINKGO_FLAGS) -focus="odo docker devfile exec command tests" tests/integration/devfile/docker
 
-# Run odo status devfile command tests
-.PHONY: test-cmd-devfile-status
-test-cmd-devfile-status:
-	ginkgo $(GINKGO_FLAGS) -focus="odo devfile status command tests" tests/integration/devfile/
-
 # Run odo devfile watch command tests
 .PHONY: test-cmd-devfile-watch
 test-cmd-devfile-watch:
 	ginkgo $(GINKGO_FLAGS) -focus="odo devfile watch command tests" tests/integration/devfile/
 
-# Run odo devfile app command tests
-.PHONY: test-cmd-devfile-app
```

Comment:
why are we removing the app integration tests?

Reply:
same as above. 
How should I proceed to get them back in? 
Can we cancel this PR and I create a new one?

Reply:
Fetching the latest master and rebasing might help.